web/src/stream_events.ts

```diff
@@ -364,17 +364,6 @@ export function mark_unsubscribed(sub: StreamSubscription): void {
     user_profile.update_user_profile_streams_list_for_users([people.my_current_user_id()]);
 }
 
-export function remove_deactivated_user_from_all_streams(user_id: number): void {
```

Comment:
The commit message from this commit should have the reasoning itself, not link to a GitHub comment.

> We changed the API in <commitID/API feature level reference> so that the server nows sends explicit `peer_remove` events before the deactivation event. 

(If we ever move off GitHub, links to individual comments will definitely stop working, so we avoid using them in permanent records like the commit history. It's also just annoying to have to click a link rather than just reading a sentence).